.github/workflows/test_migrations.yaml

```diff
@@ -0,0 +1,246 @@
+
+          sudo npm -g install npm@6.14.11
+
+          which python; python --version
+          echo npm $(npm --version)
+          echo node $(node --version)
+          nginx -v
+          firefox --version
+
+      - name: Install SkyPortal dependencies
+        run: |
+          export PYTHONPATH=$PYTHONPATH:$(pwd)
+          make dependencies
+
+          pip install black
+
+      - name: Save current Alembic head on master
+        run: |
+          # Get the head line from Alembic history output
+          head=$( PYTHONPATH=. alembic history | grep '(head)' )
+
+          # Split by space; index 2 should be latest revision
+          tokens=($head)
+          current_head=${tokens[2]}
+          echo ${current_head}
+          echo "CURRENT_HEAD=${current_head}" >> $GITHUB_ENV
+
+      - name: Checkout commit before the very first migration
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          ref: c48fa7f040aa83a494a365f80a8dffeaa0af9a1f
+          submodules: true
+
+      - name: Write SkyPortal configs
+        run: |
+          cat << EOF > config.yaml
+            database:
+              database: skyportal
+              host: localhost
+              port: 5432
+              user: skyportal
+              password: anything
+          EOF
+
+      - name: Initialize SkyPortal
+        run: |
+          # Usually, we create databases on the local machine, so
+          # `createdb` just works out of the box.  However, when doing
+          # this on a server, as in our case, we need to specify extra
+          # options.
+          #
+          # db_init should not complain if the databases exist already
+          #
+          echo "localhost:5432:*:skyportal:anything" > ~/.pgpass
+          chmod 600 ~/.pgpass
+
+          createdb -h localhost -U skyportal skyportal_test
+          psql -U skyportal -h localhost -c "GRANT ALL PRIVILEGES ON DATABASE skyportal_test TO skyportal;" skyportal_test
+
+          make db_init
+
+          pip list --format=columns
+          npm ls --depth 0
+
+          # Stamp Alembic HEAD
+          PYTHONPATH=. alembic -x config=config.yaml stamp head
+
+      - name: Test loading demo data
+        run: |
+          make run &
+          sleep 10 && make load_demo_data
+          kill %1
+
+      - name: Checkout PR branch
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          submodules: true
+          clean: false
+
+      - name: Check for migration branching
+        run: |
+          # Make sure latest revision on master is included in Alembic history
+          n_current_head=$( PYTHONPATH=. alembic history | grep -c $CURRENT_HEAD )
+          if [[ $n_current_head -eq 0 ]]
+          then
+            cat << EOF
+              The current head Alembic revision on master is not present in this branch's
+              Alembic history. Please merge in the latest commit on master.
```

Comment:
Just to be doubly clear, perhaps say "Please merge in the latest commit from master into this PR."